order-api/src/main/java/com/smoothstack/order/api/OrderApi.java

```diff
@@ -41,20 +42,18 @@
      *         or Access token is missing or invalid (status code 401)
      *         or Forbidden (status code 403)
      */
-    @ApiOperation(value = "Create order", nickname = "createOrder", notes = "Create new order, sends back checkout session data. Payment intent will be canceled in 5 minutes if not paid. (Server note: use stripe webhooks to update payment status)", response = InlineResponse200.class, authorizations = {
-        
-        @Authorization(value = "JWT")
-         }, tags={ "order", })
+    @ApiOperation(value = "Create order", nickname = "createOrder", notes = "Create new order, sends back checkout session data. Payment intent will be canceled in 5 minutes if not paid. (Server note: use stripe webhooks to update payment status)",
+            response = CreateResponse.class, tags={ "order", })
     @ApiResponses(value = { 
-        @ApiResponse(code = 200, message = "OK", response = InlineResponse200.class),
+        @ApiResponse(code = 200, message = "OK", response = CreateResponse.class),
         @ApiResponse(code = 401, message = "Access token is missing or invalid", response = String.class),
         @ApiResponse(code = 403, message = "Forbidden") })
     @PutMapping(
-        value = "/order",
+        path = "/order",
         produces = { "application/json", "application/xml" },
         consumes = { "application/json", "application/xml" }
     )
-    default ResponseEntity<InlineResponse200> createOrder(@ApiParam(value = ""  )  @Valid @RequestBody(required = false) Order order) {
+    default ResponseEntity<?> createOrder(@ApiParam(value = ""  )  @Valid @RequestBody(required = false) Order order) {
```

Comment:
Don't return generics, throw exceptions and use controller advice or @exceptionhandlers